src/main/java/spoon/javadoc/internal/JavadocBlockTag.java

```diff
@@ -50,7 +50,7 @@
 		private String keyword;
 
 		boolean hasName() {
-			return this == PARAM || this == THROWS;
+			return this == PARAM || this == THROWS || this == EXCEPTION;
```

Comment:
we have code duplication here. What about refactoring to have a single implementation?

Reply:
Yes, I've just been writing about it. It would be useful for #3033 issue too.